courses-fe/src/pages/login/index.js

```diff
@@ -1,7 +1,84 @@
-import React from "react";
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import './style.scss';
 
-const Login = () => {
-    return <div>LoginPage</div>
+class LoginPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        // reset login status
+
+        this.state = {
+            username: '',
+            password: '',
+            submitted: false
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(e) {
+        const { name, value } = e.target;
+        this.setState({ [name]: value });
+    }
+
+    handleSubmit(e) {
+        e.preventDefault();
+
+        this.setState({ submitted: true });
+        const { username, password } = this.state;
+        if (username && password) {
+            this.props.login(username, password);
```

Comment:
nici functia asta de this.props.login nu ar trebui sa fie aici pana nu o paseszi efectiv prin props...caci altfel numai da eraore...mai bine faci numai un console.log la ele si faci un redirect catre main